app/form_objects/school_search_form.rb

```diff
@@ -44,7 +44,7 @@ def select_order_state_options
 
   def csv_filename
     tokens = %w[allocations]
-    tokens << "#{array_of_urns.count}-URNs" if urns.present?
+    tokens << array_of_identifiers.count.to_s if identifiers.present?
```

Comment:
Are there any issues changing this token format?

Reply:
I've changed this back, although not technically correct I think it's the safest thing to do